<launch>
  <node pkg="xqserial_server" type="xqserial_server" name="motor_driver">
    <param name="debug_flag" value="false"/>
    <param name="bar_flag" value="true"/>
    <param name="wheel_separation" value="0.50"/>
    <param name="wheel_radius" value="0.005714"/>
    <param name="max_speed" value="13.33"/>
    <param name="port_car" value="/dev/stm32Car"/>
    <param name="port_imu" value="/dev/stm32Imu"/>
  </node>
  <node pkg="tf" type="static_transform_publisher" name="baselink_broadcaster" args="0 0 0.15 0 0 0 1 base_footprint base_link 50">
  </node>
  <node pkg="tf" type="static_transform_publisher" name="baselinkSonar_broadcaster1" args="-0.1 0.08 0.13 0.523599 0 0  base_link sonar1 50"/>
  <node pkg="tf" type="static_transform_publisher" name="baselinkSonar_broadcaster2" args="-0.1 0 0.13 0 0 0  base_link sonar2 50"/>
  <node pkg="tf" type="static_transform_publisher" name="baselinkSonar_broadcaster3" args="-0.1 -0.08 0.13 5.759586 0 0  base_link sonar3 50"/>
  <node pkg="tf" type="static_transform_publisher" name="baselinkSonar_broadcaster4" args="-0.23 0 0.13 3.1415926 0 0  base_link sonar4 50"/>
</launch>
